src/encoded/persona.py

```diff
@@ -146,3 +147,15 @@ def session(request):
         return request.session
     request.session['user_properties'] = request.embed('/current-user', as_user=userid)
     return request.session
+
+
+@view_config(route_name='impersonate-user', request_method='GET',
+             permission='impersonate')
+def impersonate_user(request):
+    """As an admin, impersonate a different user."""
+    request.session.get_csrf_token()
+    userid = request.params['userid']
+    request.session['user_properties'] = request.embed('/current-user', as_user=userid)
+    request.session['disable_persona'] = True
```

Comment:
Good point. I'm mostly on vacation this week, so will take a look at this next week if not before.